docs/content/getting-started/data-out/analyze-and-send.md

```diff
@@ -46,7 +46,7 @@ Here is how to send the data as a scalar:
 ```python
 rr.send_columns(
     "/jaw_open_state",
-    indexes=[rr.IndexColumn("frame_nr", sequence=df["frame_nr"])],
+    times=[rr.TimeColumn("frame_nr", sequence=df["frame_nr"])],
```

Comment:
In the Rust API, `times` is still called `indexes`, so we should probably unify.

Nit: `times` sounds/reads a bit weird here to me. Maybe `time_columns`, although it's a bit longer?